app/scripts/modules/amazon/src/serverGroup/details/resize/AmazonResizeServerGroupModal.tsx

```diff
@@ -0,0 +1,424 @@
+    const scalingPolicies = serverGroup.scalingPolicies || [];
+    return (
+      scalingPolicies.length && advancedMode && suspendedProcesses.every(p => p.processName !== 'AlarmNotification')
+    );
+  };
+
+  private submit = (values: IAmazonResizeServerGroupValues): void => {
+    const { min, max, desired, enforceCapacityConstraints, reason } = values;
+    const { advancedMode, interestingHealthProviderNames } = this.state;
+    const { serverGroup, application } = this.props;
+    const { asg } = serverGroup;
+    const command: IResizeJob = {
+      capacity: {
+        min: advancedMode ? min : desired,
+        max: advancedMode ? max : desired,
+        desired,
+      },
+      reason,
+      interestingHealthProviderNames,
+    };
+    if (enforceCapacityConstraints) {
+      command.constraints = {
+        capacity: {
+          min: asg.minSize,
+          max: asg.maxSize,
+          desired: asg.desiredCapacity,
+        },
+      };
+    }
+    this.state.taskMonitor.submit(() => {
+      return ReactInjector.serverGroupWriter.resizeServerGroup(serverGroup, application, command);
+    });
+  };
+
+  private renderSimpleMode(): JSX.Element {
+    const { serverGroup } = this.props;
+    const { asg } = serverGroup;
+    return (
+      <div>
+        <p>Sets min, max, and desired instance counts to the same value.</p>
+        <p>
+          To allow autoscaling, use the{' '}
+          <a className="clickable" onClick={() => this.toggleAdvancedMode()}>
+            Advanced Mode
+          </a>
+          .
+        </p>
+        <div className="form-group">
+          <div className="col-md-3 sm-label-right">Current size</div>
+          <div className="col-md-4">
+            <div className="horizontal middle">
+              <input type="number" className="NumberInput form-control" value={asg.desiredCapacity} disabled={true} />
+              <div className="sp-padding-xs-xaxis">instances</div>
+            </div>
+          </div>
+        </div>
+        <div className="form-group">
+          <div className="col-md-3 sm-label-right">Resize to</div>
+          <div className="col-md-4">
+            <div className="horizontal middle">
+              <FormikFormField name="desired" input={props => <NumberInput {...props} min={0} />} touched={true} />
+              <div className="sp-padding-xs-xaxis">instances</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  private renderAdvancedMode(): JSX.Element {
+    const { serverGroup } = this.props;
+    const { asg } = serverGroup;
+    return (
+      <div>
+        <p>Sets up autoscaling for this server group.</p>
+        <p>
+          To disable autoscaling, use the{' '}
+          <a className="clickable" onClick={() => this.toggleAdvancedMode()}>
+            Simple Mode
+          </a>
+          .
+        </p>
+        <div className="form-group bold">
+          <div className="col-md-2 col-md-offset-3">Min</div>
```

Comment:
would like to move away from bootstrap grid for forms asap.  as we discussed, there's no great non-boostrap solution for this bit though because of the min/desired/max grid